client/src/pages/Cards.js

```diff
@@ -0,0 +1,33 @@
+import React  from "react";
+import Card from "react-bootstrap/Card";
+import Button from "react-bootstrap/Button";
+const Cards=({ graduate })=>{
+    return(
+  //   <Card>
+  //   <Card.Img variant="top" src="holder.js/100px160" />
+  //   <Card.Body>
+  //     <Card.Title>{graduate.firstName+" "+graduate.surname}</Card.Title>
+  //     <Card.Text>
+
+  //   Personal Bio:{graduate.personalBio}
+  //   skills:{graduate.skills}
+  //     </Card.Text>
+  //   </Card.Body>
+  //   <Card.Footer>
+  //     <small className="text-muted">Last updated 3 mins ago</small>
+  //   </Card.Footer>
+  // </Card>
+<Card style={{ width: '18rem' }}>
+  <Card.Img variant="top" src="holder.js/100px180" />
+  <Card.Body>
+    <Card.Title>{graduate.firstName+" "+graduate.surname}</Card.Title>
+    <Card.Text>
+    Personal Bio:{graduate.personalBio}
+    skills:{graduate.skills}
+    </Card.Text>
```

Comment:
This doesn't display very nicely if you leave them in one `Card.Text` so I would split it in two. Also add some spaces after the titles, something like this:
```suggestion
    <Card.Text>
        Personal Bio: {graduate.personalBio}
    </Card.Text>
    <Card.Text>
        skills: {graduate.skills}
    </Card.Text>
```